### PR TITLE
Adding python codes to samples/python

### PR DESCRIPTION
**Pull request by adityak2920**
There are some files in samples/cpp whose python samples are not available. So, this PR is just adding "laplace.py" python code for laplace.cpp .
Added drawing.py to samples/python as it's code is not present

### CODE REVIEW

**Comment by adityak2920** [comment]
@alalek you edited the description with
 **WIP**
force_builders=Docs
test_modules=none
I am new to contributing to OpenCV, so please can you explain what it means in brief. And the build status is in waiting from the last 3 days, so can you help me in resolving the issue if any?
There should not be any whitespace error and unique branch error because I have taken care of these errors.

File: samples/python/drawing.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+    for i in range(0, 255, 2):
+        image2 = np.array(image) - i
+        cv.putText(image2, "OpenCV forever!", org, cv.FONT_HERSHEY_COMPLEX, 3, (i, i, 255), 5, lineType);
+        cv.imshow(wndname, image2);
+        if(cv.waitKey(DELAY) >= 0):
+            break
+
+    cv.waitKey(0)
+cv.destroyAllWindows()
```
Comment (alalek) [inline samples/python/drawing.py]: > cv.destroyAllWindows()

This should not be called on the module level.


Please take a look on other python samples about used skeleton.

**Comment by alalek** [comment]
This patch should [go](https://github.com/opencv/opencv/wiki/How_to_contribute#faq) into [3.4 branch](https://github.com/opencv/opencv/wiki/Branches) first. We will merge changes from 3.4 into master regularly (weekly/bi-weekly).

So, please:
- change "base" branch of this PR: master => 3.4 (use "Edit" button near PR title)
- rebase your commits from master onto 3.4 branch. For example:
    `git rebase -i --onto upstream/3.4 upstream/master`
    (check list of your commits, save and quit (Esc + "wq" + Enter)
    where `upstream` is configured by following [this GitHub guide](https://help.github.com/articles/configuring-a-remote-for-a-fork/) and fetched (`git fetch upstream`).
- push rebased commits into source branch of your fork (with `--force` option)

Note: no needs to re-open PR, apply changes "inplace".

**Comment by asmorkalov** [comment]
@alalek I rebased the branch on top of 3.4 and squashed commits.

**Comment by asmorkalov** [comment]
:+1: Tested on Ubuntu 16.04 with current 3.4 branch.

File: samples/python/laplace.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            smoothed = cv.medianBlur(frame, ksize)
+
+        # Apply Laplace function
+        laplace = cv.Laplacian(smoothed, ddepth, 5)
+        # Converting back to uint8
+        result = cv.convertScaleAbs(laplace, (sigma+1)*0.25)
+        # Display Output
+        cv.imshow("Laplace of Image", result)
+        k = cv.waitKey(30) & 0xFF
+        if k == 32:
```
Comment (alalek) [inline samples/python/laplace.py]: Why is not ESC (27)?

File: samples/python/drawing.py [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+        pt[1][2][1] = np.random.randint(y1, y2)
+        color = "%06x" % np.random.randint(0, 0xFFFFFF)
+        color = tuple(int(color[i:i+2], 16) for i in (0, 2 ,4))
+        ppt = np.asarray([*pt[0], *pt[1]])
+        cv.fillPoly(image, [ppt], color, lineType)
+        cv.imshow(wndname, image)
+        if(cv.waitKey(DELAY) >= 0):
+            break
+
+# Drwaing Circles
```
Comment (alalek) [inline samples/python/drawing.py]: typo

File: samples/python/laplace.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+'''
+    This program demonstrates Laplace point/edge detection using
+    OpenCV function Laplacian()
+    It captures from the camera of your choice: 0, 1, ... default 0
```
Comment (alalek) [inline samples/python/laplace.py]: Sample should accept command-line parameters

**Comment by VadimLevin** [comment]
@adityak2920 Thank you for your contribution.

Please split your code into functions for each drawing functionality and it is better to have the main function [reference](https://github.com/opencv/opencv/wiki/Coding_Style_Guide#python-samples) to keep this sample aligned with the others.

**Comment by VadimLevin** [comment]
Looks good to me :+1:

**Comment by adityak2920** [comment]
@alalek Please review this PR and check whether it can be merged or not.

**Comment by asmorkalov** [comment]
@alalek could you take a look and merge?

**Pull request merged**